### PR TITLE
C# Strings.Replace() entry added for review

### PR DESCRIPTION
**Pull request by PetroHubenya**
### Description

A C# entry has been created for the Strings.Replace() method. This is the first "term" entry for C# Strings, so I have created a new folder labeled "terms" under c-sharp/concepts/strings. Closes https://github.com/Codecademy/docs/issues/1840.

### Type of Change

- Adding a new entry.

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/c-sharp/concepts/strings/terms/replace/replace.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+```
+
+It produces the following output to the console:
+
+```cs
+Wrong statement is: Earth is the fourth planet from the Sun.
+Wright statement is: Earth is the third planet from the Sun.
+```
+
+Interested in helping build Docs? Read the [Contribution Guide](https://github.com/Codecademy/docs/blob/main/.github/CONTRIBUTING.md) or share your thoughts in this [feedback form](https://codecademyready.typeform.com/to/hzVIWDgz).
```
Comment (SSwiniarski) [inline content/c-sharp/concepts/strings/terms/replace/replace.md:76]: Remove this last line. It is provided by Docs automatically.

**Comment by PetroHubenya** [comment]
Dear @SSwiniarski, the [content/c-sharp/concepts/strings/terms/replace/replace.md](https://github.com/Codecademy/docs/pull/1856/commits/2a13056b85a273987038673f93b03b4ddde29c42) file is updated according to your comments.
Kindly ask you to review.

**Review by SSwiniarski — APPROVED** [review]
@PetroHubenya Looks good for a second review.

File: content/c-sharp/concepts/strings/terms/replace/replace.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+The `.Replace()` method takes the original string returns a new string with the following changes in it:
+
+* All occurances of indicated character are replaced with another one.
+* All occurances of indicated string are replaced with another one.
+
+> **Note:** If `oldChar` or `oldString` is not found in the original string, than the `.Replace()` method returns the string witout changes.
+
+> **Note:** If `newChar` or `newString` is `null`, than all occurances of `oldChar` or `oldString` are removed.
+
+> **Note:** The `.Replace()` method does not modify the original `string`. It always produces a new string.
```
Comment (caupolicandiaz) [inline content/c-sharp/concepts/strings/terms/replace/replace.md:38]:
```suggestion
> **Note:** The `.Replace()` method does not modify the original `string`, it always produces a new string.
```

File: content/c-sharp/concepts/strings/terms/replace/replace.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+// Replaces old string with new string.
+myString.Replace(string oldString, string newString);
+```
+
+The `.Replace()` method takes the original string returns a new string with the following changes in it:
+
+* All occurances of indicated character are replaced with another one.
+* All occurances of indicated string are replaced with another one.
+
+> **Note:** If `oldChar` or `oldString` is not found in the original string, than the `.Replace()` method returns the string witout changes.
```
Comment (caupolicandiaz) [inline content/c-sharp/concepts/strings/terms/replace/replace.md:34]:
```suggestion
> **Note:** If `oldChar` or `oldString` is not found in the original string, than the method returns the string without changes.
```

File: content/c-sharp/concepts/strings/terms/replace/replace.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+// Replaces old character with new character.
+myString.Replace(char oldChar, char newChar);
+
+// Replaces old string with new string.
+myString.Replace(string oldString, string newString);
+```
+
+The `.Replace()` method takes the original string returns a new string with the following changes in it:
+
+* All occurances of indicated character are replaced with another one.
```
Comment (caupolicandiaz) [inline content/c-sharp/concepts/strings/terms/replace/replace.md:31]:
```suggestion
* All occurrences of indicated character are replaced with another one.
```

File: content/c-sharp/concepts/strings/terms/replace/replace.md [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+* All occurances of indicated string are replaced with another one.
```
Comment (caupolicandiaz) [inline content/c-sharp/concepts/strings/terms/replace/replace.md:32]:
```suggestion
* All occurrences of indicated string are replaced with another one.
```

File: content/c-sharp/concepts/strings/terms/replace/replace.md [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+> **Note:** If `newChar` or `newString` is `null`, than all occurances of `oldChar` or `oldString` are removed.
+
+> **Note:** The `.Replace()` method does not modify the original `string`. It always produces a new string.
+
+## Example 1
+
+The following example takes the `string` named `oldString`, replaces all the hyphen characters `-` with comma characters `,`, and returns the new `string` named `newString`.
+
+```cs
+string oldString = "A-B-C-D-E-F-G-H-I";
```
Comment (caupolicandiaz) [inline content/c-sharp/concepts/strings/terms/replace/replace.md:45]: both code examples should include a class and main call, it should be runnable as is, like a codebyte

File: content/c-sharp/concepts/strings/terms/replace/replace.md [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
+```
+
+## Example 2
+
+The following example takes the `string` named `wrongString`, replaces the word `fourth` with the word `third`, and returns the new `string` named `rightString`.
+
+```cs
+string wrongString = "Earth is the fourth planet from the Sun.";
+string rightString = wrongString.Replace("fourth", "third");
+Console.WriteLine($"Wrong statement is: {wrongString}");
```
Comment (caupolicandiaz) [inline content/c-sharp/concepts/strings/terms/replace/replace.md:65]:
```suggestion
Console.WriteLine($"False statement: {wrongString}");
```

File: content/c-sharp/concepts/strings/terms/replace/replace.md [hunk 8]
```diff
@@ -0,0 +1,74 @@
...
+Console.WriteLine($"Right statement is: {rightString}");
```
Comment (caupolicandiaz) [inline content/c-sharp/concepts/strings/terms/replace/replace.md:66]:
```suggestion
Console.WriteLine($"True statement: {rightString}");
```

File: content/c-sharp/concepts/strings/terms/replace/replace.md [hunk 9]
```diff
@@ -0,0 +1,74 @@
...
+```
+
+It produces the following output to the console:
+
+```shell
+Wrong statement is: Earth is the fourth planet from the Sun.
```
Comment (caupolicandiaz) [inline content/c-sharp/concepts/strings/terms/replace/replace.md:72]:
```suggestion
False statement: Earth is the fourth planet from the Sun.
```

File: content/c-sharp/concepts/strings/terms/replace/replace.md [hunk 10]
```diff
@@ -0,0 +1,74 @@
...
+Right statement is: Earth is the third planet from the Sun.
```
Comment (caupolicandiaz) [inline content/c-sharp/concepts/strings/terms/replace/replace.md:73]:
```suggestion
True statement: Earth is the third planet from the Sun.
```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@PetroHubenya I've finished my initial review. Please update the entry and tag me when you've finished pushing changes. Thanks

**Comment by PetroHubenya** [comment]
Dear @caupolicandiaz, the entry is updated according your comments. Kindly ask you review ones again.

**Review by caupolicandiaz — APPROVED** [review]
@PetroHubenya Looks good. I did add a couple of edits, one issue was the sequential notes or block quotes under the syntax section. This arrangement sets off MD028/no-blanks-blockquote error. I've added comments to avoid the error, but having one note that addresses all three concerns is probably a cleaner approach.

**Comment by SSwiniarski** [comment]
@PetroHubenya congrats on the new entry! Live link is here:https://www.codecademy.com/resources/docs/c-sharp/strings/replace

You're currently listed as an anonymous contributor. If you want to be credited follow the instructions below to add your GitHub username to your Codecademy profile. If not, there is nothing more you need to do.

image

To add your GitHub username to your Codecademy profile:

1.) Add your GitHub Username to your Codecademy Profile Details, you can get there on your desktop by pressing on your profile picture in the upper right-hand corner and selecting "Profile" in the drop-down box:

image

2.) Then log out and log back into Codecademy using your GitHub login:

image

**Note:** Your email used for Codecademy will have to be listed in your GitHub profile for the login feature to properly work. If you use the same email for both GitHub and Codecademy already, then you're set.

**Pull request merged**